test/index.js

```diff
@@ -1,31 +1,31 @@
-'use strict';
+import hljs from '#hljs';
 
-const hljs = require('../build');
 hljs.debugMode(); // tests run in debug mode so errors are raised
 
 // Tests specific to the API exposed inside the hljs object.
 // Right now, that only includes tests for several common regular expressions.
-require('./api');
+import './api/index.js';
```

Comment:
Important can't look into directories using the `index` trick without a package.json?

Reply:
`index.js` lookup is deprecated in Node.js (https://nodejs.org/api/deprecations.html#DEP0151) for ESM, and not supported in the browser.